doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/98_Video_Type.md

```diff
@@ -76,12 +76,12 @@ if($videoData) {
         "height" => 480,
         "attributes" => ["class" => "video-js custom-class", "preload" => "auto", "controls" => "", "data-custom-attr" => "my-test"]
     ]);
-    $video->type = $v->getType();
-    $video->id = ($videoData instanceof Asset) ? $videoData->getId() : $videoData;
-    $video->title = $v->getTitle();
-    $video->description = $v->getDescription();
+    $video->setObjectVar('type',$v->getType());
+    $video->setObjectVar('title',$v->getTitle());
+    $video->setObjectVar('description',$v->getDescription());
+    $video->setObjectVar('id',($videoData instanceof Asset) ? $videoData->getId() : $videoData);
```

Comment:
```suggestion
    $video->setType($v->getTitle());
    $video->setDescription($v->getDescription());
    $video->setId(($videoData instanceof Asset) ? $videoData->getId() : $videoData);
```
I would also add setters for `type` & `poster` on `Pimcore\Model\Document\Editable\Video`